Blog/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
add .vscode to .gitignore

Reply:
Added .vscode to .gitignore on the feature/suggestions branch which is waiting for being merged with this one.